pkg/karmadactl/cmdinit/cert/cert_test.go

```diff
@@ -104,4 +105,30 @@ func TestGenCerts(_ *testing.T) {
 	if err := GenCerts(TestCertsTmp, etcdServerCertConfig, etcdClientCertCfg, karmadaCertCfg, apiserverCertCfg, frontProxyClientCertCfg); err != nil {
 		fmt.Println(err)
```

Comment:
Could you please help to update it to `t.Fatal(err)` and all of samhe problem from this test.

Reply:
Ok, I will replace all fmt.Println(err) with t.ror for the file concerned, but I don't quite understand what the samhe problem means, I tried to search through google, but I didn't get some possibly relevant answers

Reply:
@tiansuo114 
sorry for typo,` samhe problem` --> `same problem`